maxim/logger/crewai/utils.py

```
@@ -127,10 +130,18 @@ def stringify(obj: Any, limit: int = MAX_STR_LEN) -> str:
+            warnings.filterwarnings("ignore", category=UserWarning)
+            rep = repr(obj)
+    except RecursionError:
+        rep = f"<{type(obj).__name__}: RecursionError>"
     except Exception:
         try:
-            rep = str(obj)
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=DeprecationWarning)
+                warnings.filterwarnings("ignore", category=UserWarning)
+                rep = str(obj)
         except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)